recipes/NNPOps/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "nnpops" %}
+{% set version = "v0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mikemhenry/NNPOps/archive/refs/tags/v0.01-pre.1.tar.gz
+  #url: https://github.com/openmm/NNPOps/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Is there a reason we are using a fork as opposed to the original source?

Reply:
Just a temporary thing until a release is cut upstream.

Reply:
Seems like it has been released!
https://github.com/openmm/NNPOps/releases/tag/v0.1  !

Reply:
Which is something that upstream is onboard with https://github.com/openmm/NNPOps/issues/26#issuecomment-998203000

Reply:
I'm now using the office source